zql/zql.peg

```diff
@@ -580,6 +580,8 @@ singleCharEscape
   / "r" { RETURN("\r") }
   / "t" { RETURN("\t") }
   / "v" { RETURN("\v") }
+  / "=" { RETURN("=") }
+  / "*" { RETURN("\\*") }
```

Comment:
I think these belong in the boomWordPart rule so they aren't accepted inside quoted strings.